eq-author/src/App/questionPage/Preview/index.js

```diff
@@ -80,6 +119,24 @@ export const UnwrappedPreviewPageRoute = ({ loading, data }) => {
             <Panel dangerouslySetInnerHTML={{ __html: guidance }} />
           </Guidance>
         )}
+
+        {(!isEmpty(definitionLabel) || !isEmpty(definitionContent)) && (
+          <Details data-test="definition">
+            <DetailsTitle>
+              {definitionLabel || <Error small>Missing definition label</Error>}
+            </DetailsTitle>
+            <DetailsContent>
+              {definitionContent ? (
+                <span dangerouslySetInnerHTML={{ __html: definitionContent }} />
```

Comment:
We should probably look at sanitising on the server as currently we don't check the values users are saving so we could have one user XSS another at the moment in these preview blocks. But it doesn't block this PR as this is an existing issue.

Reply:
Lets get a Technical Improvements card raised for this.